dataprofiler/tests/profilers/test_base_column_profilers.py

```diff
@@ -272,9 +272,6 @@ def test_created_profile(self):
                 zip(self.column_profile._profiles.values(), profiler_mocks):
             self.assertIn(profile.__class__, self.profilers)
             self.assertEqual(1, profiler_mock.call_count)
-            self.assertEqual(
-                1, profiler_mock.return_value.update.call_count
-            )
```

Comment:
Can't keep a count of something which isn't called (mock the profiler) and mock's can't be pickled so this errors in multiprocess mode.